rosdep/python.yaml

```diff
@@ -1908,6 +1908,16 @@ python-gpxpy:
     zesty:
       pip:
         packages: [gpxpy]
+python-gql-pip:
```

Comment:
Our new policy is to make new rules prefixed with python3 not python as that's the new default.